tests/modules/individuals/resources/test_individual_names.py

```diff
@@ -117,3 +121,63 @@ def test_get_set_individual_names(db, flask_app_client, researcher_1):
         )
         sighting.delete_cascade()
         enc.delete_cascade()
+
+
+def test_ensure_default_name_on_individual_creation(db, flask_app_client, researcher_1):
```

Comment:
I'm now trying to use the request.addfinalizer construct to make tests more readable for the cleanup.

Reply:
Yes, I forgot about that. Will try to use in the future.